src/pages/Market/components/Item/Item.jsx

```diff
@@ -0,0 +1,28 @@
+import "./Item.css";
+import heart from "../../../../../src/assets/images/heart.svg";
+import fallback from "../../../../../src/assets/images/fallback.png";
+const Item = ({ item, size }) => {
+  const { images, name, price, favoriteCount } = item;
+  return (
+    <div className={`Item ${size}`}>
+      <img
+        src={images && images.length > 0 ? images[0] : fallback}
+        alt={name}
+        className="item-img"
+        onError={(e) => {
+          e.target.src = fallback;
+        }}
+      />
+      <div className="text-section">
+        <p className="item-name">{name}</p>
+        <p className="item-price">{price}원</p>
```

Comment:
💊 제안
금액의 경우 (,)를 통해 단위가 구분되도록 아래처럼 작성해주세요!

```suggestion
        <p className="item-price">{price.toLocalString()}원</p>
```